vector/src/main/java/im/vector/app/features/home/HomeActivity.kt

```diff
@@ -422,9 +428,17 @@ class HomeActivity :
 
     override fun onNewIntent(intent: Intent?) {
         super.onNewIntent(intent)
-        if (intent?.getParcelableExtra<HomeActivityArgs>(MvRx.KEY_ARG)?.clearNotification == true) {
+        val parcelableExtra = intent?.getParcelableExtra<HomeActivityArgs>(MvRx.KEY_ARG)
+        if (parcelableExtra?.clearNotification == true) {
             notificationDrawerManager.clearAllEvents()
         }
+        if (parcelableExtra?.inviteNotificationRoomId != null) {
+            activeSessionHolder.getSafeActiveSession()
```

Comment:
out interest, is there a convention around formatting/autoformatter? I notice this call has the chain across new lines but the [other](https://github.com/vector-im/element-android/pull/4044/files#diff-a44e22f42d574ac91aeeec4eb708fa7a9e054daa34e234da5ff92bf06d17fa5eR234) is on one line

Reply:
Formating rules files are in .idea/codeStyles/, you can read https://github.com/matrix-org/synapse/blob/master/CONTRIBUTING.md  for all details

Reply:
ah perfect, thanks!

Reply:
@ouchadam sorry wrong link https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md